### PR TITLE
Fix CI failure starting from DMD 2.085

### PR DESCRIPTION
**Pull request by Basile-z**

### CODE REVIEW

File: src/containers/unrolledlist.d [hunk 1]
```diff
@@ -565,7 +565,7 @@ private:
 			static if (BookkeepingType.sizeof < uint.sizeof)
 				immutable uint notReg = ~(cast(uint) registry);
 			else
-				immutable uint notReg = ~registry;
+				immutable uint notReg = cast(uint) (~registry);
```
Comment (Basile-z) [inline src/containers/unrolledlist.d]: @Hackerpilot , to answer to a previous review comment (#137), the two branches are not the same. The first is to fix the deprecation message related to integral promotion not done, the other fixes the issue.

File: .travis.yml [hunk 2]
```diff
@@ -27,3 +27,4 @@ script:
   - ninja -j8 -C build test -v
   - git submodule update --init --recursive
   - make -B -C test/
+  - dub test
```
Comment (Basile-z) [inline .travis.yml]: otherwise #136 is not triggered

**Comment by Basile-z** [comment]
Any objection ? I'll wait till Monday because we cant stay stuck to the fixed container version (because of #136) for ever. It's time to move forward.

**Comment by leh103** [comment]
Thanks this should allow dlangide to be built without error

**Comment by Hackerpilot** [comment]
Sorry for the lack of responses. I was on vacation last week.

**Pull request merged**